src/moj/all.mjs

```diff
@@ -133,6 +132,7 @@ export {
   ButtonMenu,
   DatePicker,
   FilterToggleButton,
+  FormValidator,
```

Comment:
I'm not sure this should be added back in... it is archived...

But then so is richTextEditor and passwordReveal... I suspect that they should have been removed when they were archived, but can't say for certain as we don't really have a process/definition for archived components and what happens to them ☹️

Reply:
Ah ok, that's fine if its a fix for something we've broken along the way. That makes sense. In which case lets leave it in here.  We'll deal with the archived-but-still-actually-there components another day - we should probably warn people we're going to deprecate them first, and then remove them later